packages/lesmis/src/components/MainMenu.js

```diff
@@ -97,7 +97,7 @@ const AdminPanelLink = () => {
   const { translate } = useI18n();
 
   return (
-    <LocaleListItemLink to="/admin" target="_blank">
+    <LocaleListItemLink to="/admin">
```

Comment:
I tend to remove these when I find them, the general UX advice on this: https://uxdesign.cc/linking-to-a-new-tab-vs-same-tab-f88b495d2187